loadgen/loadgen.cc

```diff
@@ -900,6 +914,13 @@ void PerformanceSummary::LogDetail(AsyncDetail& detail) {
     MLPERF_LOG(detail, "result_invalid_reason", recommendation);
   }
 
+  // Report number of queries
+  MLPERF_LOG(detail, "query_count", std::to_string(query_count));
```

Comment:
Just to keep the style the same, could we print it as "result_query_count" instead of "query_count" for the key like this:

```
MLPERF_LOG(detail, "result_query_count", std::to_string(query_count));
...
MLPERF_LOG(detail, "result_overlatency_query_count",
               std::to_string(overlatency_query_count));
```

Thanks!

Also, would be great if you add these two new logging fields into our logging spec: https://docs.google.com/document/d/1o2qp_wMHZ1KkmfA7h9nhrqx1kA901UOQvF_-G34tfl8/edit#heading=h.zzzo7rpxnkw (although probably no one is looking at it)